andela/mmdp_site
# MMDP-655: Restructure reset password message.

**What does this PR do?**

Restructures the message returned with instructions on how a user can reset
their password.

**description of Task to be completed?**

- [x] restructure message returned with instructions on how a user can reset
their password.

**How should this be manually tested?**

- git fetch, then git checkout to this branch `g-fix-refactor-reset-password-message-MMDP-655`
- yarn run dev-server
- request a password reset on the website
- check your email

**What are the relevant Jira issues?**

[MMDP-655](https://viisaus.atlassian.net/browse/MMDP-655)

**Screenshots (if appropriate)**

![image](https://user-images.githubusercontent.com/27225249/60493069-d39f7580-9cb4-11e9-9fd2-f93b07728634.png)

**Questions:**

None.
